Determining if the Q_WS_WIN exist failed with the following output:
Change Dir: /home/tryan/voice_analytics/pyside/PySide-1.2.4/pyside_build/py3.5-qt4.8.7-64bit-release/pyside/CMakeFiles/CMakeTmp

Run Build Command:"/usr/bin/make" "cmTC_d9400/fast"
/usr/bin/make -f CMakeFiles/cmTC_d9400.dir/build.make CMakeFiles/cmTC_d9400.dir/build
make[1]: Entering directory '/home/tryan/voice_analytics/pyside/PySide-1.2.4/pyside_build/py3.5-qt4.8.7-64bit-release/pyside/CMakeFiles/CMakeTmp'
Building CXX object CMakeFiles/cmTC_d9400.dir/CheckSymbolExists.cxx.o
/usr/bin/c++    -I/usr/include/qt4   -o CMakeFiles/cmTC_d9400.dir/CheckSymbolExists.cxx.o -c /home/tryan/voice_analytics/pyside/PySide-1.2.4/pyside_build/py3.5-qt4.8.7-64bit-release/pyside/CMakeFiles/CMakeTmp/CheckSymbolExists.cxx
/home/tryan/voice_analytics/pyside/PySide-1.2.4/pyside_build/py3.5-qt4.8.7-64bit-release/pyside/CMakeFiles/CMakeTmp/CheckSymbolExists.cxx: In function ‘int main(int, char**)’:
/home/tryan/voice_analytics/pyside/PySide-1.2.4/pyside_build/py3.5-qt4.8.7-64bit-release/pyside/CMakeFiles/CMakeTmp/CheckSymbolExists.cxx:8:19: error: ‘Q_WS_WIN’ was not declared in this scope
   return ((int*)(&Q_WS_WIN))[argc];
                   ^
CMakeFiles/cmTC_d9400.dir/build.make:65: recipe for target 'CMakeFiles/cmTC_d9400.dir/CheckSymbolExists.cxx.o' failed
make[1]: *** [CMakeFiles/cmTC_d9400.dir/CheckSymbolExists.cxx.o] Error 1
make[1]: Leaving directory '/home/tryan/voice_analytics/pyside/PySide-1.2.4/pyside_build/py3.5-qt4.8.7-64bit-release/pyside/CMakeFiles/CMakeTmp'
Makefile:126: recipe for target 'cmTC_d9400/fast' failed
make: *** [cmTC_d9400/fast] Error 2

File /home/tryan/voice_analytics/pyside/PySide-1.2.4/pyside_build/py3.5-qt4.8.7-64bit-release/pyside/CMakeFiles/CMakeTmp/CheckSymbolExists.cxx:
/* */
#include <QtCore/qglobal.h>

int main(int argc, char** argv)
{
  (void)argv;
#ifndef Q_WS_WIN
  return ((int*)(&Q_WS_WIN))[argc];
#else
  (void)argc;
  return 0;
#endif
}

Determining if the Q_WS_QWS exist failed with the following output:
Change Dir: /home/tryan/voice_analytics/pyside/PySide-1.2.4/pyside_build/py3.5-qt4.8.7-64bit-release/pyside/CMakeFiles/CMakeTmp

Run Build Command:"/usr/bin/make" "cmTC_47dfe/fast"
/usr/bin/make -f CMakeFiles/cmTC_47dfe.dir/build.make CMakeFiles/cmTC_47dfe.dir/build
make[1]: Entering directory '/home/tryan/voice_analytics/pyside/PySide-1.2.4/pyside_build/py3.5-qt4.8.7-64bit-release/pyside/CMakeFiles/CMakeTmp'
Building CXX object CMakeFiles/cmTC_47dfe.dir/CheckSymbolExists.cxx.o
/usr/bin/c++    -I/usr/include/qt4   -o CMakeFiles/cmTC_47dfe.dir/CheckSymbolExists.cxx.o -c /home/tryan/voice_analytics/pyside/PySide-1.2.4/pyside_build/py3.5-qt4.8.7-64bit-release/pyside/CMakeFiles/CMakeTmp/CheckSymbolExists.cxx
/home/tryan/voice_analytics/pyside/PySide-1.2.4/pyside_build/py3.5-qt4.8.7-64bit-release/pyside/CMakeFiles/CMakeTmp/CheckSymbolExists.cxx: In function ‘int main(int, char**)’:
/home/tryan/voice_analytics/pyside/PySide-1.2.4/pyside_build/py3.5-qt4.8.7-64bit-release/pyside/CMakeFiles/CMakeTmp/CheckSymbolExists.cxx:8:19: error: ‘Q_WS_QWS’ was not declared in this scope
   return ((int*)(&Q_WS_QWS))[argc];
                   ^
CMakeFiles/cmTC_47dfe.dir/build.make:65: recipe for target 'CMakeFiles/cmTC_47dfe.dir/CheckSymbolExists.cxx.o' failed
make[1]: *** [CMakeFiles/cmTC_47dfe.dir/CheckSymbolExists.cxx.o] Error 1
make[1]: Leaving directory '/home/tryan/voice_analytics/pyside/PySide-1.2.4/pyside_build/py3.5-qt4.8.7-64bit-release/pyside/CMakeFiles/CMakeTmp'
Makefile:126: recipe for target 'cmTC_47dfe/fast' failed
make: *** [cmTC_47dfe/fast] Error 2

File /home/tryan/voice_analytics/pyside/PySide-1.2.4/pyside_build/py3.5-qt4.8.7-64bit-release/pyside/CMakeFiles/CMakeTmp/CheckSymbolExists.cxx:
/* */
#include <QtCore/qglobal.h>

int main(int argc, char** argv)
{
  (void)argv;
#ifndef Q_WS_QWS
  return ((int*)(&Q_WS_QWS))[argc];
#else
  (void)argc;
  return 0;
#endif
}

Determining if the Q_WS_MAC exist failed with the following output:
Change Dir: /home/tryan/voice_analytics/pyside/PySide-1.2.4/pyside_build/py3.5-qt4.8.7-64bit-release/pyside/CMakeFiles/CMakeTmp

Run Build Command:"/usr/bin/make" "cmTC_4ce1c/fast"
/usr/bin/make -f CMakeFiles/cmTC_4ce1c.dir/build.make CMakeFiles/cmTC_4ce1c.dir/build
make[1]: Entering directory '/home/tryan/voice_analytics/pyside/PySide-1.2.4/pyside_build/py3.5-qt4.8.7-64bit-release/pyside/CMakeFiles/CMakeTmp'
Building CXX object CMakeFiles/cmTC_4ce1c.dir/CheckSymbolExists.cxx.o
/usr/bin/c++    -I/usr/include/qt4   -o CMakeFiles/cmTC_4ce1c.dir/CheckSymbolExists.cxx.o -c /home/tryan/voice_analytics/pyside/PySide-1.2.4/pyside_build/py3.5-qt4.8.7-64bit-release/pyside/CMakeFiles/CMakeTmp/CheckSymbolExists.cxx
/home/tryan/voice_analytics/pyside/PySide-1.2.4/pyside_build/py3.5-qt4.8.7-64bit-release/pyside/CMakeFiles/CMakeTmp/CheckSymbolExists.cxx: In function ‘int main(int, char**)’:
/home/tryan/voice_analytics/pyside/PySide-1.2.4/pyside_build/py3.5-qt4.8.7-64bit-release/pyside/CMakeFiles/CMakeTmp/CheckSymbolExists.cxx:8:19: error: ‘Q_WS_MAC’ was not declared in this scope
   return ((int*)(&Q_WS_MAC))[argc];
                   ^
CMakeFiles/cmTC_4ce1c.dir/build.make:65: recipe for target 'CMakeFiles/cmTC_4ce1c.dir/CheckSymbolExists.cxx.o' failed
make[1]: *** [CMakeFiles/cmTC_4ce1c.dir/CheckSymbolExists.cxx.o] Error 1
make[1]: Leaving directory '/home/tryan/voice_analytics/pyside/PySide-1.2.4/pyside_build/py3.5-qt4.8.7-64bit-release/pyside/CMakeFiles/CMakeTmp'
Makefile:126: recipe for target 'cmTC_4ce1c/fast' failed
make: *** [cmTC_4ce1c/fast] Error 2

File /home/tryan/voice_analytics/pyside/PySide-1.2.4/pyside_build/py3.5-qt4.8.7-64bit-release/pyside/CMakeFiles/CMakeTmp/CheckSymbolExists.cxx:
/* */
#include <QtCore/qglobal.h>

int main(int argc, char** argv)
{
  (void)argv;
#ifndef Q_WS_MAC
  return ((int*)(&Q_WS_MAC))[argc];
#else
  (void)argc;
  return 0;
#endif
}

